EVGEN/AliGenExtFile.cxx

```diff
@@ -276,6 +290,32 @@ void AliGenExtFile::Generate()
 	fNprimaries++;
     } // track loop
 
+    // User Trigger
+    if(fSetUserTrig){
+      AliStack *stack = AliRunLoader::Instance()->Stack();
+      if(!fUserTrigger(stack)){continue;}
+    }
+
+    // Base multiplicity trigger
+    if(fSetMultTrig){
+      AliInfo(Form("mult cut : %i",fMultCut));
+      AliStack *stack = AliRunLoader::Instance()->Stack();
+      if(!MultiplicityTrigger(stack)){
+        consecutiveDiscardedEvents++;
+        if(consecutiveDiscardedEvents>10){AliFatal("More than 10 events discarded consequently");}
```

Comment:
Why is it needed to count how many events are discarded and not allow more than 10? I guess if you request a very large multiplicity in pp events it is possible to have such situation

Reply:
Indeed you are right! I added this line when I was testing different multiplicity cuts in order to avoid runnung even if no event is found. Are you siggesting to remove it or to simply increase the threshold for discarded events?

Reply:
At least not a hardcoded number. Devote a dedicated data member and put as default a large number (99999?) and a print in case this number is reached.

Reply:
Thanks! I have addd the data member as you suggested in my new commit.